tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Check if OpenTripPlanner service exists
-  stat: path=/etc/init/otp.conf
+  stat: path=/lib/systemd/system/otp.service
```

Comment:
Although `systemd` picks up files in this location and `/etc/systemd/system`, I think using `/etc/systemd/system` would be better. My understanding is that package specific service configurations go in `/lib`, and user service configurations go in `/etc`.